dev-support/examples/dp-tf/build.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euxo pipefail
+
+SUBMARINE_VERSION=0.6.0-SNAPSHOT
+SUBMARINE_IMAGE_NAME="distribution:${SUBMARINE_VERSION}"
```

Comment:
```suggestion
SUBMARINE_IMAGE_NAME="apache/submarine:distribution-${SUBMARINE_VERSION}"
```
You can update this [file](https://github.com/apache/submarine/blob/master/.github/workflows/deploy_docker_images.yml), GA will publish this image to Apache docker hub